libs/langchain-scripts/package.json

```diff
@@ -22,14 +22,14 @@
   "scripts": {
     "build": "yarn clean && yarn turbo:command build:internal --filter=@langchain/scripts",
     "build:internal": "tsc --project ./tsconfig.build.json && yarn move:artifacts && yarn build:generated",
-    "move:artifacts": "rimraf dist && mkdir -p dist && mv dist_build/* dist/",
+    "move:artifacts": "rimraf dist && node -e \"const fs = require('fs'); fs.mkdirSync('dist', {recursive: true}); fs.cpSync('dist_build', 'dist', {recursive: true});\"",
```

Comment:
Since we are using `rimraf` , let's be consistent and also use `mkdirp` from NPM for this.